docs/mcd.md

```diff
@@ -0,0 +1,329 @@
+      <option value="0" label="Binary"/>
+      <option value="2" label="HTTP"/>
+    </selection-group>
+ 
+    <selection-group>
+      <conditions>
+        <condition field="authentication-type" value="1"/>
+      </conditions>
+      <option value="1" label="SASL"/>
+      <option value="2" label="HTTP"/>
+    </selection-group>
+ 
+    <selection-group>
+      <conditions>
+        <condition field="authentication-type" value="2"/>
+      </conditions>
+      <option value="1" label="SASL"/>
+    </selection-group>
+   
+    <selection-group>
+      <conditions>
+        <condition field="authentication-type" value="3"/>
+      </conditions>
+      <option value="0" label="Binary"/>
+      <option value="1" label="SASL"/>
+      <option value="2" label="HTTP"/>
+    </selection-group>
+ 
+    <selection-group>
+      <conditions>
+        <condition field="authentication-type" value="6"/>
+      </conditions>
+      <option value="2" label="HTTP"/>
+    </selection-group>
+ 
+  </field>
+ 
+  <field name="kerberos-realm" label="Realm" category="authentication" value-type="string">
+    <conditions>
+      <condition field="authentication-type" value="1"/>
+    </conditions>
+  </field>
+ 
+  <field name="kerberos-host" label="Host FQDN" category="authentication" value-type="string">
+    <conditions>
+      <condition field="authentication-type" value="1"/>
+    </conditions>
+  </field>
+ 
+  <field name="kerberos-service" label="Service Name" category="authentication" value-type="string">
+    <conditions>
+      <condition field="authentication-type" value="1"/>
+    </conditions>
+  </field>
+ 
+  <field name="username" label="Username" category="authentication" value-type="string">
+    <conditions>
+      <condition field="authentication-type" value="2"/>
+      <condition field="authentication-type" value="3"/>
+      <condition field="authentication-type" value="6"/>
+    </conditions>
+  </field>
+ 
+  <field name="password" label="Password" category="authentication" value-type="string" secure="true">
+    <conditions>
+      <condition field="authentication-type" value="3"/>
+      <condition field="authentication-type" value="6"/>
+    </conditions>
+  </field>
+ 
+  <field name="http-path" label="HTTP Path" category="authentication" value-type="string">
+    <conditions>
+      <condition field="transport-type" value="2"/>
+    </conditions>
+  </field>
+ 
+  <field name="SSL" label="Require SSL" value-type="boolean" default-value="">
+    <boolean-options>
+      <false-value value="" />
+      <true-value value="1" />
+    </boolean-options>
+  </field>
+ 
+  <field name="TrustedCerts" label="Custom SSL certificate file" value-type="file">
+    <conditions>
+      <condition field="SSL" value="1"/>
+    </conditions>
+  </field>
+ 
+  <field name="AutoReconnect" label="" value-type="string" default-value="0" editable="false" />
+ 
+</connection-fields>
+```
+
+
+# The Connection Metadata File
+
+**TODO** link to xsd?
```

Comment:
The XSD is in the dev-2020.2 branch right now https://github.com/tableau/connector-plugin-sdk/blob/dev-2020.2/connector-packager/connector_packager/xsd_validator.py#L25, which sould be merged to master branch once 2020.2 is out and this documentation should be made public

Reply:
Added link.